extras/beast/unit_test/suite_list.hpp

```diff
@@ -18,34 +18,41 @@
 namespace beast {
 namespace unit_test {
 
-/** A container of test suites. */
+/// A container of test suites.
 class suite_list
     : public detail::const_container <std::set <suite_info>>
 {
 private:
-#ifndef NDEBUG
-    std::unordered_set <std::string> names_;
-    std::unordered_set <std::type_index> classes_;
+#if 0//#ifndef NDEBUG
```

Comment:
It should be put back, I commented it out to see if it fixed a crashing bug and forgot to put it back